packages/mandelbrot/README.md

```diff
@@ -2,22 +2,33 @@
 
 The default web UI theme for Fractal.
 
-See the Fractal [documentation](http://fractal.build/guide) for details on configuration and usage.
+[![NPM Version](https://img.shields.io/npm/v/@frctl/mandelbrot.svg?style=flat-square)](https://www.npmjs.com/package/@frctl/mandelbrot)
+
+See the Fractal [documentation](https://fractal.build/guide) for details on [configuration and usage](https://fractal.build/guide/web/default-theme.html).
 
 In order to get a locale aware date for the 'last updated' text, install the
 [full-icu](https://github.com/unicode-org/full-icu-npm) module.
 
-## Customize theme labels
+## Development
+
+[Follow these instructions](https://github.com/frctl/fractal#development) to setup the development environement, then go to the Mandelbrot package directory:
+
+```
+cd packages/mandelbrot
+```
 
-Some theme-specific labels can be overridden via config options:
+In there, run:
 
 ```
-const theme = require('@frctl/mandelbrot')({
-    labels: {
```

Comment:
I think this is not documented on the docs site, maybe open an issue there if we're removing this info from here?

Reply:
https://github.com/frctl/fractal-docs/issues/57